src/nx-migrate.ts

```diff
@@ -20,8 +23,7 @@ export async function migrate(keepMigrationsFile: boolean): Promise<void> {
       }
     }
   )
-  // sometimes migrations change packages without installing them, so naivly install dependencies here again
-  await exec('npm install --package-lock-only')
+  await exec(inputs.installCommand)
```

Comment:
We could consider adding a second input to allow customization of the installation command. However, it might be sufficient to use the default installCommand again. With the latest versions of Yarn, caching is highly efficient, making subsequent installs very fast when there are minimal changes. Still, this approach might be slower for some dependency management tools.

Reply:
hi thanks for the PR! I think its quite heavy to have a one-for-all install command. Sometimes we need `npm ci` sometimes we need the `--package-lock-only` flag. What do you think about the following: name the input something like `installStrategy` which can be `"yarn" | "npm"` and for now you just check if it is npm, than you can use the existing command, if it is yarn, you can just use yarn, or something specific yarn offers. What do you think?